webpack.config.js

```diff
@@ -3,9 +3,7 @@ const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
 const TerserJSPlugin = require('terser-webpack-plugin');
 const FixStyleOnlyEntriesPlugin = require("webpack-fix-style-only-entries");
-const RemovePlugin = require('remove-files-webpack-plugin');
```

Comment:
Should we remove this from `package.json` too? It doesn't seem to be used anywhere else 🤔